docs/appkit/flutter/core/options.mdx

```diff
@@ -15,7 +15,12 @@ final _appKitModal = ReownAppKitModal(
     ),
   ),
   // enableAnalytics: true, // OPTIONAL - null by default
-  // enableEmail: true, // OPTIONAL - false by default
+  // siweConfig: SIWEConfig(...), // OPTIONAL - null by default
+  // featuresConfig: FeaturesConfig(
+  //   email: true, // OPTIONAL - false by default
+  //   socials: [...], // OPTIONAL - empty by default
```

Comment:
IMO we should add some true example here for socials and also it would be great to have a list of possible logins.  
Is there a place that said that the order is respected ?

Reply:
You can see that in the given sections, `siweConfig:` explanation is in **One-Click Auth + SIWE** section, `featuresConfig:` explanation is in **Onboarding > Email & Socials** section.

This here is just a placeholder to show every option but I'm open to suggestions.

Reply:
Can I resolve and merge?

Reply:
dale noma'